invisible_cities/cities/components.py

```diff
@@ -957,7 +979,7 @@ def calculate_and_save_buffers(buffer_length    : float        ,
                                out  = "buffers"                            )
 
     saved_buffers = "buffers" if order_sensors is None else "ordered_buffers"
-    max_subevt    =  max_time // buffer_length + 1
+    max_subevt    =  math.ceil(max_time / buffer_length)
```

Comment:
It is correct, but since `math` library is not used elsewhere, I suggest to use `numpy`, such that:
`int(np.ceil(max_time / buffer_length)`.

Reply:
I think in this case I prefer `math` since it also does type conversion (in Python 3)

Reply:
Okay, so I'll keep `math.ceil()`, right?

Reply:
yes